tsconfig.base.json

```diff
@@ -4,7 +4,7 @@
   "compilerOptions": {
     "target": "es5",
     "module": "commonjs",
-    "lib": ["dom", "es2017"],
+    "lib": ["dom", "es2017", "esnext"],
```

Comment:
I think that `"es2017"` is unnecessary 😄

Reply:
it is neccesary for `flatMap`